pyproject.toml

```diff
@@ -1,18 +1,32 @@
+[build-system]
+requires = [
+    "hatchling",
+    "hatch-vcs",
+]
+build-backend = "hatchling.build"
+
 [project]
 name = "bmaptool"
 description = "BMAP tools"
 dynamic = ["version"]
 dependencies = [
+    "defusedxml",
     "gpg >= 1.10.0",
+    'importlib-metadata>=4.6; python_version < "3.10"',
 ]
-required-python = ">= 3.8"
+requires-python = ">=3.8"
 authors = [
+    {name = "Artem Bityutskiy", email = "dedekind1@gmail.com"},
+]
```

Comment:
I believe this was removed on purpose

Reply:
It was out of sync with the README but I can take it out of the project file.